integration/spark/src/main/scala/org/apache/spark/sql/listeners/MVListeners.scala

```diff
@@ -67,7 +67,7 @@ object AlterDataMaptableCompactionPostListener extends OperationEventListener {
     val allDataMapSchemas = DataMapStoreManager.getInstance
       .getDataMapSchemasOfTable(carbonTable).asScala
       .filter(dataMapSchema => null != dataMapSchema.getRelationIdentifier &&
-                               !dataMapSchema.isIndexDataMap)
+                               !dataMapSchema.isIndex)
```

Comment:
Can delete this class

Reply:
Yes, will be delete when clean the old mv implementation